openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -2549,7 +2549,7 @@ def test_linesearch(self):
 
         model.nonlinear_solver = om.NewtonSolver()
         model.nonlinear_solver.options['maxiter'] = 3
-        # model.nonlinear_solver.options['solve_subsystems'] = True
```

Comment:
Why are we changing solve_subsystems here from true to false?

Reply:
Found that it was commented out originally. Uncommented and changed